pynitrokey/fido2/__init__.py

```diff
@@ -23,13 +24,20 @@ def newdel(self):
     usb._objfinalizer._AutoFinalizedObjectBase.__del__ = newdel
 
 
-def _UDP_InternalPlatformSwitch(funcname, *args, **kwargs):
+def _UDP_InternalPlatformSwitch(
+    funcname: Callable, *args: Tuple, **kwargs: Dict
+) -> None:
     if funcname == "__init__":
         return HidOverUDP(*args, **kwargs)
     return getattr(HidOverUDP, funcname)(*args, **kwargs)
```

Comment:
I think the `None` return value is wrong here.

Reply:
jup, but this is dead code anyways, udp has not been working for a long time now, solo1 just recently brought it back again (https://github.com/solokeys/solo1-cli/tree/main/solo/fido2) I would suggest to pull this is needed somewhere